.github/labeler.yml

```diff
@@ -0,0 +1,13 @@
+area/csi:
+- csi/**/*
+area/ci:
+- .travis
+- .github/**/*
+area/osd-cmd:
+- osd/**/*
+area/api:
+- api/**/*
+area/sdk-server:
+- api/server/sdk/**/*
+area/pkg:
+- pkg/**/*
```

Comment:
the pkg area doesn't add much value since it's the kitchen sink. Can be removed unless you see a use case

Reply:
Removed